frontend/lib/badge-url.spec.js

```diff
@@ -59,6 +59,28 @@ describe('Badge URL functions', function() {
     ].join(''))
   });
 
+  test(dynamicBadgeUrl, () => {
+    const yamlUrl = 'http://example.com/foo.yaml';
+    const query = '$.bar';
+    const prefix = 'value: ';
```

Comment:
Aren't these three tests of `dynamicBadgeUrl` testing exactly the same thing? Maybe we should strip out two of them.

Reply:
Good point. Let's remove two of them.